tests/common.py

```diff
@@ -775,6 +775,7 @@ async def json(**kwargs):
                 "X-RateLimit-Remaining": "999",
                 "X-RateLimit-Reset": "999",
                 "Content-Type": "application/json",
+                "Etag": "321",
```

Comment:
Is it OK to always include this here, or should it be more targeted?

Reply:
I think that is fine, if needed to adjust later we can always add an override for it.